spec/datetimeformat.html

```diff
@@ -95,7 +94,7 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
           1. Set _formatOptions_.[[&lt;_prop_&gt;]] to _value_.
         1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
         1. Let _hcDefault_ be _dataLocaleData_.[[hourCycle]].
-        1. Let _hc_ be _dateTimeFormat_.[[HourCycle]].
+        1. Let _hc_ be _r_.[[hc]].
```

Comment:
This delocalizes references to _r_, which hinders readability for me. But I think it could work if HourCycle were addressed **before** TimeZone, e.g.
```
        …
        1. Let _r_ be ResolveLocale(%DateTimeFormat%.[[AvailableLocales]], _requestedLocales_, _opt_, %DateTimeFormat%.[[RelevantExtensionKeys]], _localeData_).
        1. Set _dateTimeFormat_.[[Locale]] to _r_.[[locale]].
        1. Set _dateTimeFormat_.[[Calendar]] to _r_.[[ca]].
        1. Set _dateTimeFormat_.[[NumberingSystem]] to _r_.[[nu]].
        1. Let _hc_ be _r_.[[hc]].
        1. Let _dataLocale_ be _r_.[[dataLocale]].
        1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
        1. Let _hcDefault_ be _dataLocaleData_.[[hourCycle]].
        1. If _hour12_ is *true*, then
          1. If _hcDefault_ is *"h11"* or *"h23"*, set _hc_ to *"h11"*. Otherwise, set _hc_ to *"h12"*.
        1. Else if _hour12_ is *false*, then
          1. If _hcDefault_ is *"h11"* or *"h23"*, set _hc_ to *"h23"*. Otherwise, set _hc_ to *"h24"*.
        1. Else,
          1. Assert: _hour12_ is *undefined*.
          1. If _hc_ is *null*, set _hc_ to _hcDefault_.
        1. Set _dateTimeFormat_.[[HourCycle]] to _hc_.
        1. Let _timeZone_ be ? Get(_options_, *"timeZone"*).
        …
        1. Set _dateTimeFormat_.[[TimeZone]] to _timeZone_.
        1. Let _formatOptions_ be a new Record.
        1. For each row of <emu-xref href="#table-datetimeformat-components"></emu-xref>, except the header row, in table order, do
          …
        1. Set _formatOptions_.[[hourCycle]] to _dateTimeFormat_.[[HourCycle]].
        …
```